src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp

```diff
@@ -99,6 +99,17 @@ inline fs::path belgianPkcs11ModulePath()
     return "/usr/lib/x86_64-linux-gnu/libbeidpkcs11.so.0";
 #endif
 }
+	
+inline fs::path czechPkcs11ModulePath()
+{
+#ifdef _WIN32
+    return system32Path() / L"eopproxyp11.dll";
+#elif defined __APPLE__
+    return "/usr/local/lib/eOPCZE/libeopproxyp11.dylib"; 
+#else // Linux
```

Comment:
Would you mind adding preemptive OpenBSD support such that future ports for these card drivers won't require further changes to work?
```
#elif defined __OpenBSD__
    return "/usr/local/lib/libeopproxyp11.so";
```

That would go in line with #59.